modules/dnn/src/layers/blank_layer.cpp

```diff
@@ -161,8 +161,7 @@ class BlankLayerImpl CV_FINAL : public BlankLayer
                                         const std::vector<Ptr<BackendNode> >& nodes) CV_OVERRIDE
     {
         auto ieInpNode = nodes[0].dynamicCast<InfEngineNgraphNode>()->node;
-        ov::OutputVector inp{ieInpNode};
-        auto blank = std::make_shared<ov::op::v0::Concat>(inp, 0);
+        auto blank = std::make_shared<ov::op::v1::ConvertLike>(ieInpNode, ieInpNode);
```

Comment:
Concat doesn't work here when with 0d tensors

Reply:
Please add cv_assert with comment then.

Reply:
I mean, I've changed Concat to ConvertLike because ConvertLike should work fine in all cases